ClangPowerTools/ClangPowerToolsShared/Tooling/v1/psClang/get-llvm-helper.ps1

```diff
@@ -75,6 +97,18 @@ Function Ensure-LLVMTool-IsPresent([Parameter(Mandatory = $true)][string] $clang
       Write-Verbose "Downloading $clangToolWeNeed $kCptGithubLlvmVersion ..."
       # grab ready-to-use LLVM binaries from Github
       Invoke-WebRequest -Uri $clangCompilerWebPath -OutFile $llvmLiteToolPath
+      # download css file if needed tool is clang-doc.exe
+      if($clangToolWeNeed -eq $kClangDoc)
+      {
+        $parentDirLite = (get-item $llvmLiteDir ).parent.FullName
```

Comment:
Please don't forget to annotate the types of variables, in order to save yourself later headaches regarding types used.